frontend/src/pages/GraphicalSelector/GraphicalSelector.tsx

```diff
@@ -251,6 +259,7 @@ const GraphicalSelector = () => {
         </S.GraphPlaygroundWrapper>
         {sidebar && (
           <S.SidebarWrapper>
+            <Tabs items={items} />
             {course ? <div>{course.code} - {course.title}</div> : 'No course selected'}
```

Comment:
feel free to remove this line here
`{course ? <div>{course.code} - {course.title}</div> : 'No course selected'}`